src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1927,7 +1927,15 @@ public ResponseEntity<String> evaluateFilter(
             @Parameter(description = "Node uuid") @PathVariable("nodeUuid") UUID nodeUuid,
             @Parameter(description = "Should get in upstream built node ?") @RequestParam(value = "inUpstreamBuiltParentNode", required = false, defaultValue = "false") boolean inUpstreamBuiltParentNode,
             @RequestBody String filter) {
-
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.evaluateFilter(studyUuid, nodeUuid, inUpstreamBuiltParentNode, filter));
     }
+
+    @GetMapping(value = "/studies/{studyUuid}/filter/export")
```

Comment:
"/studies/{studyUuid}/filters/{filterUuid}/export "  ?

Reply:
Yes, I was hesitating because a filter is not directly a sub-resource of a study.
But you're right, it's better to use the Path.
And I want to replace export by elements, this is easier to understand.